twisted/scripts/_twistd_unix.py

```diff
@@ -198,6 +199,38 @@ def preApplication(self):
         self.oldstderr = sys.stderr
 
 
+    def _formatException(self, exception):
+        """
+        Format the C{exception} in preparation for writing to the
+        status pipe.  This does the right thing on Python 2 if the
+        exception's message is Unicode, and in all cases limits the
+        length of the message, prior to encoding, to 100 codepoints.
+
+        This means the return message may be longer than 100 bytes.
```

Comment:
I think that 100 byte is just a way to prevent excessive output..

the docstring can be a bit vague and inform that messages are truncated at about 100 characters :)